<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEvent</class>
 <widget class="QDialog" name="AddEvent">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Background: #232123;</string>
  </property>
  <widget class="QLabel" name="TitleLbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;, sans-serif;
text-transform: uppercase;
outline: 0;
width: 100%;
border: 0;
padding: 10px;
color: #AAA;
font-size: 12px;</string>
   </property>
   <property name="text">
    <string>Ajouter Evènement</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;, sans-serif;
text-transform: uppercase;
outline: 0;
width: 100%;
border: 0;
padding: 10px;
font-size: 12px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="hourStartSpin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:&quot;Segoe UI&quot;, sans-serif;
background: #777;
</string>
   </property>
   <property name="maximum">
    <number>23</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="minStartSpin">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:&quot;Segoe UI&quot;, sans-serif;
background: #777;
</string>
   </property>
   <property name="maximum">
    <number>55</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="durationSpin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:&quot;Segoe UI&quot;, sans-serif;
background: #777;</string>
   </property>
   <property name="minimum">
    <number>30</number>
   </property>
   <property name="maximum">
    <number>750</number>
   </property>
   <property name="singleStep">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="startHourLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;, sans-serif;
text-transform: uppercase;
outline: 0;
width: 100%;
border: 0;
padding: 7px;
color: #AAA;
font-size: 12px;</string>
   </property>
   <property name="text">
    <string>Heure de début :</string>
   </property>
  </widget>
  <widget class="QLabel" name="durationLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;, sans-serif;
text-transform: uppercase;
outline: 0;
width: 100%;
border: 0;
padding: 10px;
color: #AAA;
font-size: 12px;</string>
   </property>
   <property name="text">
    <string>Durée :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="validateBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>194</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font-family:&quot;Segoe UI&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #5cb85c;
	border: 0;
	padding: 7px;
	color: #FFFFFF;
	font-size: 14px;
	border-radius: 7px;
}

QPushButton:hover
{
font-family:&quot;Segoe UI&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #449d44;
	border: 0;
	padding: 7px;
	color: #FFFFFF;
	font-size: 14px;
	border-radius: 7px;
}</string>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>194</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font-family:&quot;Segoe UI&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #5cb85c;
	border: 0;
	padding: 7px;
	color: #FFFFFF;
	font-size: 14px;
	border-radius: 7px;
}

QPushButton:hover
{
font-family:&quot;Segoe UI&quot;, sans-serif;
	text-transform: uppercase;
	outline: 0;
	background: #449d44;
	border: 0;
	padding: 7px;
	color: #FFFFFF;
	font-size: 14px;
	border-radius: 7px;
}</string>
   </property>
   <property name="text">
    <string>Annuler</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>20</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;, sans-serif;
text-transform: uppercase;
outline: 0;
width: 100%;
border: 0;
padding: 7px;
color: #AAA;
font-size: 12px;</string>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="modeComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;, sans-serif;
background: #777;
text-transform: uppercase;
font-size: 12px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>55</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI,sans-serif</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: &quot;Segoe UI&quot;, sans-serif;
text-transform: uppercase;
outline: 0;
width: 100%;
border: 0;
padding: 7px;
color: #AAA;
font-size: 12px;</string>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
